1237-registrator_worklist/1238-creator_opens_my_messages.feature

```diff
@@ -1,12 +1,15 @@
 Feature: Creator opens My Messages
 
     @1238
-    Scenario: Creator opens My Messages
+    Scenario Outline: Creator opens My Messages
         Given that the user is logged in as Creator
         When they click the menu item My Messages
         Then they see the My Messages page
-        And they see a list of <MessageType> with fields <MessageTypeFields>
+        And they see a list of <MessageType> with fields:
+            | Publication title |
+            | Date              |
+            | Request status    |
```

Comment:
Skjønner ikke hva dette er (det kommer ikke frem at det er gjort en request, så hva skal request status bety?)

Reply:
for DoiRequest vil det være "Godkjent", "Avvist", eller "Under behandling". kan sikkert legge dette til i eksempler også, siden det nok vil variere basert på type